client/src/app/posts/components/post-list/post-list.component.html

```diff
@@ -0,0 +1,15 @@
+<ng-container *ngIf="isloaded; else loading">
```

Comment:
bad approach, do not do in that way, you rerender each inner components!!!
Just add spinner block to the page, add transparent overlay and position center. On HTTP request - show/hide spinner, but do not remove other components from page